configs/common/packages.yaml

```diff
@@ -243,10 +243,10 @@ packages:
   # https://github.com/JCSDA/spack-stack/issues/1276
   py-matplotlib:
     require: '@3.7.4'
-  # Variant ~mpi not working for latest py-netcdf4
-  # https://github.com/spack/spack/issues/47652
+  # https://github.com/Unidata/netcdf4-python/issues/1389
```

Comment:
This change was cherry-picked from the JCSDA release/1.9.0 branch and finds its way back into JCSDA develop before the release/1.9.0 branch is merged back. It should disappear after we merge the spack release/1.9.0 into spack-stack-dev and the spack-stack 1.9.0 branch into develop.